Presentation.Web/Controllers/API/V2/External/ItContracts/ItContractV2Controller.cs

```diff
@@ -135,6 +135,7 @@ public IHttpActionResult GetItContract([NonEmptyGuid] Guid contractUuid)
         /// <returns></returns>
         [HttpPost]
         [Route]
+        [SwaggerResponseRemoveDefaults]
```

Comment:
FYI - this removes the default `200` which takes over the "top" stage (the 201 will otherwise be in the "additional response section"